pkgs/applications/networking/cluster/argocd/default.nix

```diff
@@ -2,18 +2,18 @@
 
 buildGoModule rec {
   pname = "argocd";
-  version = "2.2.5";
-  commit = "8f981ccfcf942a9eb00bc466649f8499ba0455f5";
-  tag = "v${version}";
+  version = "2.3.0";
 
   src = fetchFromGitHub {
     owner = "argoproj";
     repo = "argo-cd";
-    rev = tag;
-    sha256 = "sha256-wSvDoRHV4BObRL8lEpHt9oGXNB06LXdIYasRYqmM5QA=";
+    rev = "v${version}";
+    sha256 = "sha256-/dfCUENrg4i4WL+hD5WK7OcDIILQSMLPsN717QvvZyY=";
   };
 
-  vendorSha256 = "sha256-BVhts+gOM6nhcR1lkFzy7OJnainLXw5YdeseBBRF2xE=";
+  vendorSha256 = "sha256-uA9sOMuVHKRRhSGoLyoKcUYU6NxtprVUITvVC+tot1g=";
+
+  subPackages = [ "cmd" ];
```

Comment:
As you have restricted the packages availability, you need to remove unused packages from `postInstall` as well.

Reply:
Even with cmd, there is a few packages there:

```
argocd
argocd-application-controller
argocd-dex
argocd-repo-server
argocd-server
argocd-util
```